src/sst/core/bootshared.cc

```diff
@@ -121,23 +124,30 @@ void boot_sst_executable(const char* binary, const int verbose, char* argv[], co
             break;
 
         case ENAMETOOLONG:
-            fprintf(stderr, "Unable to launch SST, the path to the executable exceeds the operating system maximum. Try installing to a shorter path.\n");
+            fprintf(
+                stderr, "Unable to launch SST, the path to the executable exceeds the operating system maximum. Try "
```

Comment:
Why are there new lines here between the function and `stderr`?

Reply:
The flag is set to put in a newline after the paren if the function call won't all fit on one line.  This lines up all the parameters better for long function names (i.e. all the variables aren't shoved way to the right).  There are other ways to set these flags, but this produced the most readable result.